library/src/shared/pipes/asset/asset.pipe.ts

```diff
@@ -108,10 +111,7 @@ export class AssetPipe implements PipeTransform, OnDestroy {
             Constants.MIN_FRACTION_DIGITS.toString() +
             '-' +
             asset.decimals.toString();
-          return (
-            asset.symbol +
-            formatNumber(tradeUnit.toNumber(), this.locale, digitsInfo)
-          );
+          return formatNumber(tradeUnit.toNumber(), this.locale, digitsInfo);
```

Comment:
Why did we refactor the symbol out of the pipe? Won't this cause issues with consistency across the app?

Reply:
I had already refactored code across the app to reflect this change. The pipe takes a whole asset model so the calling component always has the symbol, which we can easily add to the template. I did this to try and reduce the amount of changes but it seems I need to add a case to the pipe to handle localization properly anyways.

Reply:
I've opted towards using Angular's built in currency pipe for numbers that don't need to be converted, just formatted. The changes to the asset pipe have been reverted.